src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -730,6 +753,48 @@ subroutine ocn_gm_init(err)!{{{
       call mpas_pool_get_config(ocnConfigs, 'config_Redi_surface_layer_tapering_extent',config_Redi_surface_layer_tapering_extent)
       call mpas_pool_get_config(ocnConfigs, 'config_Redi_bottom_layer_tapering_depth',config_Redi_bottom_layer_tapering_depth)
 
+
+      block => domain % blocklist
+      do while (associated(block))
+         call mpas_pool_get_subpool(block % structs, 'mesh', meshPool)
+         call mpas_pool_get_subpool(block % structs, 'diagnostics', diagnosticsPool)
+         call mpas_pool_get_dimension(meshPool, 'nEdges', nEdges)
+         call mpas_pool_get_array(meshPool, 'cellsOnEdge', cellsOnEdge)
+         call mpas_pool_get_array(diagnosticsPool, 'gmBolusKappa', gmBolusKappa)
+         call mpas_pool_get_array(meshPool, 'areaCell', areaCell)
+
+         ! initialize Bolus kappa array
+         if (config_GM_Bolus_kappa_function == 'constant') then
+            !$omp do schedule(runtime)
+            do iEdge = 1, nEdges
+               gmBolusKappa(iEdge) = config_standardGM_tracer_kappa
+            end do
+            !$omp end do
+         else if (config_GM_Bolus_kappa_function == 'ramp') then
+            sqrtPiInv = 1.0_RKIND / sqrt(pii)
+            !$omp do schedule(runtime)
+            do iEdge = 1, nEdges
+               avgCellDiameter = sqrtPiInv*(sqrt(areaCell(cellsOnEdge(1,iEdge))) + sqrt(areaCell(cellsOnEdge(2,iEdge))))
```

Comment:
I guess this seems slightly weird to me, since our cells aren't circular.  But it's probably a fractional difference.

Reply:
That is a common way to approximate a cell diameter.  The circle would be between the inscribed and circumscribed circles.
![image](https://user-images.githubusercontent.com/4036816/47314376-b16e3780-d5fe-11e8-80f7-41b660431ff5.png)